gitman/settings.py

```diff
@@ -6,6 +6,7 @@
 
 # Cache settings
 CACHE = os.path.expanduser(os.getenv('GITMAN_CACHE', "~/.gitcache"))
+CACHE_DISABLE = True if os.getenv('GITMAN_CACHE_DISABLE') != None else False
```

Comment:
This can simply be: `bool(os.getenv('GITMAN_CACHE_DISABLE'))`

Reply:
Does not produce the same results when env variable is specified (without value -> empty string)

Have tried the following with specified env variable:
```python
print(bool(os.getenv('GITMAN_CACHE_DISABLE')))
print(True if os.getenv('GITMAN_CACHE_DISABLE') != None else False)
```
Output:
```
False
True
```

Reply:
What do you mean by "env variable is specified without value"? The convention for environmental flags is to `export GITMAN_CACHE_DISABLE=true` or `export GITMAN_CACHE_DISABLE=1`.

Reply:
Of course, this is more correct... 
Then I think the best implementation is:

```python
def str2bool(v):
  return v.lower() in ("true", "1")

# Cache settings
CACHE_DISABLE = str2bool(os.getenv('GITMAN_CACHE_DISABLE', 'False'))
``` 
According to [SO](https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python):

> 
> Be cautious when using the following:
> 
> bool("foo")
> True
> bool("")
> False
> Empty strings evaluate to False, but everything else evaluates to True. So this should not be used for any kind of parsing purposes.

Reply:
I don't really want the value of the flag to matter, but it _does_ need to be set to a value.

`bool(os.getenv('GITMAN_CACHE_DISABLE'))` is sufficent.

Reply:
I see...